vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -4,12 +4,12 @@ sidebarDepth: 2
 
 # Entando Customer Portal 
 ## Overview
-The Entando Customer Portal enables an organization to quickly provide a modern, self-service customer-facing application for management of subscriptions. It includes a lightweight integration to Jira Service Management for access to service tickets and a Role Based Access Control (RBAC) design for granting users varying levels of access.
+The Entando Customer Portal (CP) is a modern, self-service customer-facing application for managing subscriptions, providing organizations with a quick ready-made solution. It includes a lightweight integration to Jira Service Management to administer service tickets and implement Role Based Access Controls (RBACs).
```

Comment:
There are two commas in the first sentence.  The adjectives do not need commas or 'and' because they are tiered, quick describes ready made just like self-service describes customer-facing. It's like bright yellow sun, you don't have to say bright, yellow sun or bright and yellow sun.

Reply:
kk that makes sense. however, that sentence is currently jam-packed with hyphenated, tiered adjs. it needs to be broken up somehow to be more easily readable; i was thinking commas could be a way to do that.

Reply:
also, why have a comma after modern?